tests/RoyalmailTest.php

```diff
@@ -24,72 +28,307 @@ public function setUp()
+                    'insuranceValue' => '20',
+                    'shippingMethodNameClean' => 'Second Class: Large Letter',
+                    'size' => '',
+                ),
+            10 =>
+                array (
+                    'shippingMethodName' => 'UK_STANDARD_SECOND_CLASS_SMALL_PARCEL',
+                    'minimumWeight' => '0.001',
+                    'maximumWeight' => '1.000',
+                    'methodPrice' => '2.80',
+                    'insuranceValue' => '20',
+                    'shippingMethodNameClean' => 'Second Class: Small Parcel',
+                    'size' => 'SMALL',
+                ),
+            11 =>
+                array (
+                    'shippingMethodName' => 'UK_STANDARD_SECOND_CLASS_MEDIUM_PARCEL',
+                    'minimumWeight' => '0.001',
+                    'maximumWeight' => '1.000',
+                    'methodPrice' => '4.89',
+                    'insuranceValue' => '20',
+                    'shippingMethodNameClean' => 'Second Class: Medium Parcel',
+                    'size' => 'MEDIUM',
+                ),
+            12 =>
+                array (
+                    'shippingMethodName' => 'UK_CONFIRMED_ROYAL_MAIL_SIGNED_FOR_FIRST_CLASS_LETTER',
+                    'minimumWeight' => '0.001',
+                    'maximumWeight' => '0.100',
+                    'methodPrice' => '1.73',
+                    'insuranceValue' => '50',
+                    'shippingMethodNameClean' => 'Signed For: First Class Letter',
+                    'size' => '',
+                ),
+            13 =>
+                array (
+                    'shippingMethodName' => 'UK_CONFIRMED_ROYAL_MAIL_SIGNED_FOR_FIRST_CLASS_LARGE_LETTER',
+                    'minimumWeight' => '0.001',
+                    'maximumWeight' => '0.100',
+                    'methodPrice' => '2.05',
+                    'insuranceValue' => '50',
+                    'shippingMethodNameClean' => 'Signed For: First Class Large Letter',
+                    'size' => '',
+                ),
+            14 =>
+                array (
+                    'shippingMethodName' => 'UK_CONFIRMED_ROYAL_MAIL_SIGNED_FOR_FIRST_CLASS_SMALL_PARCEL',
+                    'minimumWeight' => '0.001',
+                    'maximumWeight' => '1.000',
+                    'methodPrice' => '4.40',
+                    'insuranceValue' => '50',
+                    'shippingMethodNameClean' => 'Signed For: First Small Parcel',
+                    'size' => 'SMALL',
+                ),
+            15 =>
+                array (
+                    'shippingMethodName' => 'UK_CONFIRMED_ROYAL_MAIL_SIGNED_FOR_FIRST_CLASS_MEDIUM_PARCEL',
+                    'minimumWeight' => '0.001',
+                    'maximumWeight' => '1.000',
+                    'methodPrice' => '6.75',
+                    'insuranceValue' => '50',
+                    'shippingMethodNameClean' => 'Signed For: First Medium Parcel',
```

Comment:
My recent PR changed the name of this and the previous method to have "Signed For: First Class Medium Parcel" and "Signed For: First Class Small Parcel".  Does this need to be updated?